Realm/RLMRealm.h

```diff
@@ -169,7 +171,7 @@
 /**
  The RLMSchema used by this RLMRealm.
  */
-@property (nonatomic, readonly) RLMSchema *schema;
+@property (nonatomic, readonly, null_unspecified) RLMSchema *schema;
```

Comment:
The commit message for this change (d80e58bd262e6fc9464da5d6541077a7e62d0d22) explains why this is `null_unspecified` rather than `nonnull`.